### DISCUSSION TITLE
RSS support?

### DISCUSSION THREAD

**Discussion by JohnMadden123456** [civitai/civitai#304] [Ideas]
Is it possible to have RSS stream of all published models/lora/everything else? No filter, just everything that is published should be fine.

**Comment by WASasquatch** [comment]
Yeah a RSS dump of everything uploaded, it's type, description, featured image, etc, would be great.

**Comment by Bricklou** [comment]
I won't be against a RSS support because it's a bit hard to follow all the available models publication/update on the discord.

**Reply by WASasquatch** [reply]
I honestly miss the days of checking my feed lists right in my browser from anywhere. few sites support it still. Luckily the ones that do I use a lot like 3D models

**Comment by TomKranenburg** [comment]
Just tossing my hat into the ring here.

RSS would be great for keeping up with model updates. At the moment I don't think I have any way of finding out when that happens.

**Comment by NaokiSato102** [comment]
I believe that RSS support is a feature that should naturally be included. If, for some reason—perhaps a religious one—you find it necessary to reject this, then an appropriate alternative should be provided.

Those who desire RSS support are lamenting the inadequacy of the official notification system, and they are hoping for some form of update detection that can substitute for their own manual checking. Have you not realized that a system which only broadcasts whether an update has occurred, as opposed to requiring full-scale communication just to verify updates, could actually help alleviate the traffic issues that have been a constant problem on this site?

**Reply by WASasquatch** [reply]
The API is much more mature now. You can just load up models section, or images, filter as needed, and sort by date.

**Reply by Subr1ata** [reply]
> The API is much more mature now. You can just load up models section, or images, filter as needed, and sort by date. 

I am new to this project, can you help me to understand this project ?
